cpp/cmake/common.cmake

```diff
@@ -1,52 +1,58 @@
-cmake_minimum_required(VERSION 3.24)
+cmake_minimum_required(VERSION 3.21)
 
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+if (WIN32)
+  set(Ice_NUGET_NAME "zeroc.ice.v143")
+  set(Ice_NUGET_DIR "${CMAKE_CURRENT_LIST_DIR}/packages/${Ice_NUGET_NAME}")
```

Comment:
I guessing that I can use a custom NuGet build by just installing it to this location, which would skip the install below.